content/de/index.md

```diff
@@ -40,7 +40,7 @@ function Counter() {
 
 
 <section class="home-section">
-  <img src="/home/metal.svg" alt="metal">
+  <img src="/home/metal.svg" alt="metal" loading="lazy" decoding="async" width="54" height="54">
```

Comment:
`decoding="async"` is the only real addition here, lazy & the sizes have been added to the English doc ages ago but hasn't quite made it to all the translations. Figured I'd fix it now.